src/utils/update/trust.model.fetch.ts

```diff
@@ -226,6 +226,10 @@ export const mapCorporateTrusteeData = (trustee: CorporateTrusteeData, trust: Tr
     mapHistoricalCorporateTrusteeData(trustee, trust);
     return;
   }
+  if (trusteeRoleType === RoleWithinTrustType.BENEFICIARY) {
+    mapBeneficiaryCorporateTrusteeData(trustee, trust);
+    return;
+  }
```

Comment:
This code has caused an issue.  See https://companieshouse.atlassian.net/browse/UAR-1607 . A new method should not be needed. Corporate trustees who are beneficiaries are already being mapped. This code prevents that mapping.